### DISCUSSION TITLE
The official Safari app has been abandoned by the developer.

### DISCUSSION THREAD

**Discussion by WoodpeckerBaby** [darkreader/darkreader#11208] [General]
It's outdated and no longer works on more than half of the websites. Not even on the google search result page. 

Please coordinate and have a different developer take over the Safari project, considering that it's a paid app.

**Comment by ImBIOS** [comment]
I'd love if we move it from app store to this GitHub release, so It can be free maybe. But when it's paid, I expect a better experience to have.

**Reply by WoodpeckerBaby** [reply]
Same. I emailed the developer listed on App Store twice over the past year, never got a response. His website is outdated and still says 2021.

**Comment by stackptr** [comment]
Perhaps look into Noir. While I've been using it on iOS devices for a while, I just found that there is a macOS extension app for purchase that seems an adequate replacement for Dark Reader. I came to this repo mainly to check if there was work underway to sync settings between websites, and Noir already has this functionality. It was also updated last week, so appears to be actively maintained.

**Reply by WoodpeckerBaby** [reply]
I'm using Dark Night.
